packagers/pnpm-workspaces-default-output/test.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 
-FILE0=node_modules/.prisma/client/index.js
 FILE1=sub-project-1/node_modules/.prisma/client/index.js
 FILE2=sub-project-2/node_modules/.prisma/client/index.js
 
 set -eux
 
-cd workspace
+cd workspace/sub-project-1
+FILE0=$(node -e "console.log(path.join(require.resolve('@prisma/client/package.json'), '..', '..', '..', '.prisma'))")
```

Comment:
What's the resolved path of this?

Reply:
Something like
`/home/millsp/Work/e2e-tests/packagers/pnpm-workspaces-default-output/workspace/node_modules/.pnpm/@prisma+client@3.12.0-integration-fix-yarn1-workspaces.2_663dcaeee949835cfab1a843c306e085/node_modules/.prisma`